### PR TITLE
Best practices for Docker file in /web.

### PR DESCRIPTION
**Pull request by aserputov**
## Issue This PR Addresses

Fixes: #1668 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [x] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/web/app/.dockerignore [hunk 1]
```diff
@@ -4,6 +4,7 @@ npm-debug.log
 Dockerfile
 .git
 .gitignore
-
 .next
 out
+ .env*
+ env*
```
Comment (AmasiaNalbandian) [inline src/web/app/.dockerignore:10]: Do we have `.env` in this directory? Not sure if this is as a pre-caution?

Comment (aserputov) [reply src/web/app/.dockerignore:10]: Yes, it's a precaution

Comment (AmasiaNalbandian) [reply src/web/app/.dockerignore:10]: I really wonder if we should include it then... I can't find anything that suggests which is better/not?

**Review by Kevan-Y — CHANGES_REQUESTED** [review]
- Can the `base` stage use a specific version of node (e.g: ` node:16`)?

- Would be nice to use `pnpm` for installation. 

- I don't think `tini` would be needed since we can declared `tini:true` in docker-compose

**Comment by menghif** [comment]
> * Can the `base` stage use a specific version of node (e.g: ` node:16`)?
> 
>     * Would be nice to use `pnpm` for installation.
> 
>     * I don't think `tini` would be needed since we can declared `tini:true` in docker-compose

I implemented these changes to https://github.com/Seneca-CDOT/telescope/pull/3367.

@aserputov can you review my PR?

**Comment by TueeNguyen** [comment]
Closing because #3367 addresses the same problem

**Pull request closed without merging**